Interop/Indexer/src/main/kotlin/org/jetbrains/kotlin/native/interop/indexer/MacroConstants.kt

```diff
@@ -110,7 +114,7 @@ private fun reparseWithCodeSnippet(library: NativeLibrary,
         val codeSnippetLines = when (library.language) {
             // Note: __auto_type is a GNU extension which is supported by clang.
             Language.C, Language.OBJECTIVE_C -> listOf(
-                    "const __auto_type KNI_INDEXER_VARIABLE = $name;"
+                    "void kni_indexer_function() { __auto_type KNI_INDEXER_VARIABLE = $name; }"
```

Comment:
Is it required to define a wrapping function?

Reply:
Looks like it's required. Without the wrapping function the variable is global and we cannot use a function to initialize it. So we cannot expand macros like `#define FOO foo()`.

Reply:
Btw, is `errno` supported now? On macOS it is declared like
```
#define errno (*__error())
```

Reply:
Yes, it's supported now.